src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's unit number in the address book.
+ */
+public class Unit {
+
+    private String unitNumber;
+
+    public Unit(String unitNumber) {
+        this.unitNumber = unitNumber;
+    }
+
+    public String getUnitNumber() {
+        return unitNumber;
+    }
+
+    @Override
+    public String toString() {
+        return unitNumber;
+    }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
I am not sure if this comparison works fully well.
What happens if the other is an instance of `Unit` but not `this`?